core/src/graph/key_manager.rs

```diff
@@ -176,12 +179,27 @@ impl PublicKeyProvider for PublicKeyManager {
 
 		Ok(())
 	}
+
+	fn filter_users_without_keys(&self, dsnp_user_ids: Vec<DsnpUserId>) -> Vec<DsnpUserId> {
```

Comment:
nit: maybe 'get' or 'find' would be more appropriate naming than 'filter'

Reply:
will change